src/assets/wise5/classroomMonitor/dataExport/abstract-export.component.ts

```diff
@@ -86,4 +86,26 @@ export abstract class AbstractExportComponent {
     }
     return selectedNodes;
   }
+
+  /**
+   * @param users An array of user objects. Each user object contains an id and name.
+   * @returns {object} An object that contains key/value pairs. The key is userIdX
+   * or studentNameX where X is an integer. The values are the corresponding actual
+   * values of user id and student name.
+   */
+  extractUserIDsAndStudentNames(users: any[]): any {
```

Comment:
Use [Record](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) to specify types of key and value on the returned object?


```suggestion
  extractUserIDsAndStudentNames(users: any[]): Record<string, number | string>  {
```